src/com/magento/idea/magento2plugin/util/RegExUtil.java

```diff
@@ -68,6 +68,9 @@
 
         public static final String ACL_RESOURCE_ID
                 = "^([A-Z]+[a-zA-Z0-9]{1,}){1,}_[A-Z]+[A-Z0-9a-z]{1,}::[A-Za-z_0-9]{1,}$";
+
```

Comment:
Why we need 2 completely the same rules? Let's just give the unified name like XML_IDENTIFIER.

Reply:
@VitaliyBoyko I've fixed the code by recommendation, thanks.